src/Packetery/Module/WebRequestClient.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Class WebRequestClient
+ *
+ * @package Packetery
+ */
+
+declare( strict_types=1 );
+
+namespace Packetery\Module;
+
+use Packetery\Core\Interfaces\IWebRequestClient;
+use Packetery\Module\Exception\WebRequestException;
+
+/**
+ * Class WebRequestClient
+ *
+ * @package Packetery
+ */
+class WebRequestClient implements IWebRequestClient {
+
+	private const TIMEOUT = 30;
+
+	/**
+	 * POST.
+	 *
+	 * @param string               $uri Target URI.
+	 * @param array<string, mixed> $options Options.
+	 */
+	public function post( string $uri, array $options ): string {
+		$resultResponse = wp_remote_post( $uri, $options );
+
+		return wp_remote_retrieve_body( $resultResponse );
+	}
+
+	/**
+	 * GET.
+	 *
+	 * @param string $apiUrl Target URL.
+	 * @param string $apiKey API key.
+	 *
+	 * @return string
+	 * @throws WebRequestException WebRequestException.
+	 */
+	public function get( string $apiUrl, string $apiKey ): string {
```

Comment:
Metoda přijímá stejné parametry, jako funkce sprintf, ale nikde to není napsáno - phpdocy jsou nedostatečné. Navrhuji rozdělit na dvě metody - get (obecná) a nějaká specializovaná, která má tyto parametry.

Reply:
Rozdeleno.